packages/dev/biome-config/configs/biome-base.jsonc

```diff
@@ -8,6 +8,9 @@
       "enabled": true,
       "lineWidth": 100,
       "indentStyle": "space"
+    },
+    "parser": {
+      "allowComments": true
```

Comment:
Most of the tools in JS ecosystem allow comments within json files. We also have quite a few json files with those.